cypress/integration/lpa/donors/donor.spec.js

```diff
@@ -1,35 +1,44 @@
-// describe("Create Donor", { tags: ["@lpa", "@smoke-journey"] }, () => {
-//   beforeEach(() => {
-//     cy.loginAs("Case Manager");
-//     cy.visit("/lpa/home");
-//   });
-
-//   it("should create a new donor", function () {
-//     cy.intercept({ method: "GET", url: "/*/v1/assignees/*/tasks*" }).as(
-//       "tasksRequest"
-//     );
-//     cy.wait("@tasksRequest");
-
-//     cy.get("uib-tab-heading[id=Timeline]").contains("Timeline").click();
-//     cy.contains("Create Donor").click();
-
-//     cy.frameLoaded(".action-widget-content iframe");
-//     cy.enter(".action-widget-content iframe").then((getBody) => {
-//       getBody().find("#f-firstname").type("Spongebob");
-//       getBody().find("#f-surname").type("Squarepants");
-//       getBody().find("button[type=submit]").click();
-
-//       getBody().contains(/Person (\d+(-|)){3} was created/);
-//       getBody().contains("View donor").click();
-//     });
-
-//     cy.get(".timeline .timeline-event", { timeout: 10000 });
-//     cy.contains(".timeline-event", "Person (Create / Edit)").should(
-//       "contain",
-//       "Spongebob Squarepants"
-//     );
-//   });
-// });
+describe("Create Donor", { tags: ["@lpa", "@smoke-journey"] }, () => {
+  beforeEach(() => {
+    cy.loginAs("Case Manager");
+    cy.visit("/lpa/home");
+  });
+
+  it("should create a new donor", function () {
+    cy.intercept({ method: "GET", url: "/*/v1/assignees/*/tasks*" }).as(
+      "tasksRequest"
+    );
+    cy.wait("@tasksRequest");
+
+    cy.get("uib-tab-heading[id=Timeline]").contains("Timeline").click();
+    cy.contains("Create Donor").click();
+
+    cy.frameLoaded(".action-widget-content iframe");
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().find("#f-firstname").type("Spongebob");
+      getBody().find("#f-surname").type("Squarepants");
+      getBody().find("button[type=submit]").click();
+    });
+
+    cy.wait(1000);
```

Comment:
As with another test, this is necessary because the Cypress doesn't naturally wait for page loads inside an iframe.